engines/scumm/he/moonbase/moonbase.h

```diff
@@ -42,9 +42,6 @@ class Moonbase {
 	void deallocateArray(int array);
 	int callScummFunction(int scriptNumber, int paramCount,...);
 
-	void blitDistortion(byte *bufferData, const int bufferWidth, const int bufferHeight, const int bufferPitch,
```

Comment:
Why? 🤔

Reply:
It's not implemented

Reply:
Right, sorry, I forgot I moved this declaration to `wiz_he.h`